sagemaker-clarify/fairness_and_explainability/bias_analysis_config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "dataset_type": "text/csv",
+  "headers": ["Target", "Age", "Workclass", "fnlwgt", "Education", "Education-Num", "Marital Status", "Occupation", "Relationship", "Ethnic group", "Sex", "Capital Gain", "Capital Loss", "Hours per week", "Country"],
+  "label": "Target",
+  "label_values_or_threshold": [1],
+  "facet": [{
+    "name_or_index": "Sex",
+    "value_or_threshold": [0]
+  }],
+  "group_variable": "Age",
+  "probability_threshold": 0.8,
+  "methods": {
+    "pre_training_bias": {
+      "methods": "all"
+    },
+    "post_training_bias": {
+      "methods": "all"
+    },
+    "report": {
+      "name": "report",
+      "title": "Analysis Report"
+    }
+  },
+  "predictor": {
+    "model_name": "DEMO-clarify-model-06-01-2023-09-28-42",
```

Comment:
I have updated the model name to a static value, going with the second approach would mean using S3Uploader/SagemakerSDK in second section which we are trying to avoid.